### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: './model/SAC_1_20_real_point_q1'

### ISSUE THREAD

**Issue by banana16314** [rxlqn/Reinforcement-Learning-Based-Parcel-Singulation-with-Variable-State-Space-Dimension#1]
Hi, bro. Thanks for your excellent work.
When I run python window.py, the error will occur.
Could u provide the model file? 
Best wishes!

**Comment by AliHaider20** [comment]
This is because the model path is by default set to the upper path. So what you can do is change the path of the model in code or rename your model as SAC_1_20_real_point_q1 and keep it in the "model" folder. The model folder must be in your program folder.

**Comment by banana16314** [comment]
Thanks for your reply！Bro, now i am running SAC.py, it looks like this:
![image](https://user-images.githubusercontent.com/9431699/125764126-d70542d4-5cf6-4509-a03c-0cbad7c9aa89.png)
Episode:  42 | Episode Reward:  355.04528883444857
...
Is it running well?

**Comment by AliHaider20** [comment]
Yeah, I guess it is running well. The reason you're seeing this is because the verbosity is True but I guess default language is not english. There might be an option to switch the language in the code. I haven't downloaded it that's why can't help.
